src/arch/xtensa/Kconfig

```diff
@@ -17,6 +17,12 @@ config SMP
 	help
 	  Indicates that architecture uses multiple cores
 
+config HAVE_SLAVE_CORE_ROM
+	bool "Have slave core rom"
+	default y
+	help
+	  Indicates that slave core have rom, alternate reset vector is needed if not
+
```

Comment:
Isn't it just for TGL? APL has it f.e.?

Reply:
Currently every multi core platform have slave core rom except TGL